app/resources/views/pages/location_search/spreadsheet.blade.php

```diff
@@ -15,7 +15,16 @@ class="form-control" id="distance"
 			<button type="button" id="updateRadius" class="btn btn-primary">Update</button>
 	</div>
 	@if (count($locations) === 0)
-		No location found matching the specified keywords
+		<h3>No location found matching the specified keywords</h3>
+			@if ($base_user->isSignedIn())
+				<a class="btn btn-default" href="/add-location">Add New Location</a>
+			@else
+				<a class="btn btn-default" href="/sign-in">Sign In</a>
+				<h4>or</h4>
+				<a class="btn btn-default" href="/sign-up">Sign Up</a>
```

Comment:
There is a similar problem with /sign-up.  I think it should be /signup.  Test it to be sure.